src/gc/handletable.cpp

```diff
@@ -313,6 +313,14 @@ OBJECTHANDLE HndCreateHandle(HHANDLETABLE hTable, uint32_t uType, OBJECTREF obje
     }
 #endif // _DEBUG && !FEATURE_REDHAWK
 
+    // If we are creating a variable-strength handle, verify that the
+    // requested variable handle type is valid.
+    if (uType == HNDTYPE_VARIABLE && !IS_VALID_VHT_VALUE(lExtraInfo))
```

Comment:
@jkotas Not sure I understood your comment correctly. Do we want this function to succeed in release builds if `lExtraInfo` is not a valid variable handle type?